src/main/java/edu/harvard/iq/dataverse/pidproviders/PidProvider.java

```diff
@@ -1,19 +1,16 @@
-package edu.harvard.iq.dataverse;
```

Comment:
I see why you did not move `PidProvider` and `PidProviderFactory` to the SPI package - we would need to deal with a data model in there, too. Good choice to prepare as much as possible so this next step is well-prepared.